sdk/src/rest/types.ts

```diff
@@ -0,0 +1,13 @@
+export interface RestCommand<_Input extends object, _Output extends object | unknown, _Schema extends object> {
+	(): RestRequestOptions;
+}
+
+export type HttpMethod = 'GET' | 'SEARCH' | 'POST' | 'PUT' | 'PATCH' | 'DELETE';
```

Comment:
Lets standardize on PascalCase, even for abbreviations, so `HttpMethod` instead of `HTTPMethod`